pandas/core/common.py

```diff
@@ -473,3 +477,16 @@ def f(x):
         f = mapper
 
     return f
+
+
+def convert_to_list_like(
+    values: Union[Scalar, Iterable, AnyArrayLike]
+) -> Union[List, AnyArrayLike]:
```

Comment:
Had to rewrite this a bit to get it to type check

Reply:
@dsaxton i noticed you used ABCIndex instead of ABCIndexClass, was that intentional?

Reply:
I don't think so, definitely change if you feel it's not correct